.github/workflows/cut-release-branch.yml

```diff
@@ -108,7 +108,7 @@ jobs:
       - name: "Create New Release Branch"
         run: |
           git checkout -b ${{ inputs.new_branch_name }}
-      
+
```

Comment:
The below step (that you didn't write) is misleading.  I believe lines 117 and 118 can just be removed.  All we're really doing here is pushing up a new branch, not actually making any changes.